odoo/fields.py

```diff
@@ -978,6 +978,8 @@ def determine_value(self, record):
                 self.compute_value(record)
             else:
                 recs = record._in_cache_without(self)
+                if len(recs) > PREFETCH_MAX:
```

Comment:
Nice! So this would affect all computed fields, not just related/inherited fields, right?
Shouldn't this be fixed as of 8.0, as it looks like the issue was already [there](https://github.com/odoo/odoo/blob/8.0/openerp/fields.py#L935-L937)?